### DISCUSSION TITLE
Why not a footer component?

### DISCUSSION THREAD

**Discussion by fyardlest1** [twbs/bootstrap#39071] [Q&A]
I always asked myself this question: Why after so many years, why after so many versions, bootstrap still doesn't have a Footer Component just like the Navbar component? That would be so great.

**Comment by patkon** [comment]
Just out of interest. What do you think the footer component should contain?

Somehow - maybe just my feeling - all footers are the same. A little color contrast to the rest, a few links and the copyright information.

**Reply by fyardlest1** [reply]
You maybe right, but I don't think all footer are the same, just like all navbar are not the same even when we use the navbar component. But I was thinking about a pre-build component just like the navbar, a ready to use similar to some specific website like Amazon, Udemy that we can customize without wasting time creating our own just like we do with the navbar component. But everyone has their own opinion.

**Reply by mdo** [reply]
Like these? https://getbootstrap.com/docs/5.3/examples/footers/

**Comment by julien-deramond** [comment]
There's no specific Footer component in Bootstrap because they are simpler than navbars and buildable without extra CSS like it can be seen in our [Footers examples](https://getbootstrap.com/docs/5.3/examples/footers/).

**Reply by fyardlest1** [reply]
That's what I am talking about, I think those Footers examples can be just footer components that we can take and customize just like we do with the Navbar component, quite simple.
